.ci_support/linux_64_.yaml

```diff
@@ -1,3 +1,5 @@
+cdt_name:
+- cos6
```

Comment:
```suggestion
```

Reply:
You shouldn't have edited this file, it is auto generated

Reply:
I have reverted my edit